website/src/categories/components/textarea/code.md

```diff
@@ -3,33 +3,53 @@ title: Textarea code
 tags: code
 APIdescription: Component has a range of properties to define the experience in different use cases.
 eleventyNavigation:
-  parent: Textarea
+  parent: Text area
   key: TextareaCode
 ---
-<section>
+<section class="no-heading">
+
+<div class="ds-example">
+  <sl-textarea
+    id="subject"
+    aria-label="My favourite subject"
+    value="My favourite subject is English. It is an important language to learn."
+    style="width: 250px;"
+  ></sl-textarea>
+</div>
 
-## Textarea code
+<div class="ds-code">
 
-Textarea examples
+  ```html
+    <sl-textarea id="subject" aria-label="My favourite subject" value="My favourite subject is..."></sl-textarea>
+  ```
 
-Lorem ipsum dolor sit amet. Consectetur adipiscing elit. Etiam eu turpis molestie, dictum est a, mattis tellus. Sed dignissim, metus nec fringilla accumsan, risus sem sollicitudin lacus, ut interdum tellus elit sed risus. Maecenas eget condimentum velit, sit amet feugiat lectus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Praesent auctor purus luctus enim egestas, ac scelerisque ante pulvinar. Donec ut rhoncus ex. Suspendisse ac rhoncus nisl, eu tempor urna. Curabitur vel bibendum lorem. Morbi convallis convallis diam sit amet lacinia. Aliquam in elementum tellus.
+</div>
 
 </section>
 
 <section>
 
-## Code
+## Installation
 
-Lorem ipsum dolor sit amet. Consectetur adipiscing elit. Etiam eu turpis molestie, dictum est a, mattis tellus. Sed dignissim, metus nec fringilla accumsan, risus sem sollicitudin lacus, ut interdum tellus elit sed risus. Maecenas eget condimentum velit, sit amet feugiat lectus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Praesent auctor purus luctus enim egestas, ac scelerisque ante pulvinar. Donec ut rhoncus ex. Suspendisse ac rhoncus nisl, eu tempor urna. Curabitur vel bibendum lorem. Morbi convallis convallis diam sit amet lacinia. Aliquam in elementum tellus.
+With npm
 
-</section>
+<div class="ds-code">
 
-{% include "../component-table.njk" %}
+  ```bash
```

Comment:
Can we create a web component that we can reuse for all these "Installation" sections?

Reply:
We can discuss it in the future, right now we just want to go really fast with documentation.